cmd_test.go

```diff
@@ -0,0 +1,39 @@
+package gps
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+)
+
+func mkTestCmd(iterations int) *monitoredCmd {
+	return newMonitoredCmd(
+		exec.Command("go", "run", "./_testdata/src/cmd/echosleep.go", "-n", fmt.Sprint(iterations)),
+		200*time.Millisecond,
+	)
+}
+
+func TestMonitoredCmd(t *testing.T) {
+	cmd := mkTestCmd(2)
+	err := cmd.run()
+	if err != nil {
+		t.Errorf("expected command not to fail:", err)
+	}
+
+	expectedOutput := "foo\nfoo\n"
+	if cmd.buf.buf.String() != expectedOutput {
+		t.Errorf("expected output %s to be %s", cmd.buf.buf.String(), expectedOutput)
```

Comment:
So, I have [a convention I follow with got/want dumping of testing output](https://github.com/sdboyer/gps/blob/366fea271c8afc71712409c8e7c529ed5a21be00/rootdata_test.go#L27). I realize this is entirely my own standard, but as long as the tests are in gps, it's preferable to keep things consistent. Do you mind converting to that?